detective.gemspec

```diff
@@ -22,8 +22,8 @@ Gem::Specification.new do |spec|
   spec.files = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = 'bin'
+  spec.executables   = %w[console detective setup]
```

Comment:
Should console and setup really be part of the gem? I also think I tried this when I wanted to write a patch but when I set ```bindir``` to ```bin``` and would only name ```detective``` it'd still include both the other bins so this line might even be redundant *if* you want to include them.